toolbox/index.md

```diff
@@ -109,7 +109,7 @@ Browser: Google Chrome
 ![Google Chrome](google-chrome.png){:width="350"}
 </figure>
 
-**What it is.** D’uh, I bet you know what a browser is, because you’re reading this in one right now. So what is this doing in the Toolbox? Well, typically a web application is expected to work in any browser, but in the interest of keeping things simple, we test your application in only one browser. This is a somewhat realistic constraint because modern browsers are becoming more alike.
+**What is it.** Duh, I bet you know what a browser is, because you’re reading this in one right now. So what is this doing in the Toolbox? Well, typically a web application is expected to work in any browser, but in the interest of keeping things simple, we test your application in only one browser. This is a somewhat realistic constraint because modern browsers are becoming more alike.
```

Comment:
Can you please just drop the “duh” altogether? I know I wrote it, but I don’t like it.